spec/components/grid-spec.js

```diff
@@ -4,6 +4,7 @@ import createComponent from '../helpers/createComponent.js';
 import Grid from '../../src/renderer/components/grid.jsx';
 import generatorsList from '../fixtures/generators-list.js';
 
+
```

Comment:
well, we should really try to normalize it, right now if you browse through the renderer files you'll see that half of them are using double line break after imports and half of them are not

the double line break is a personal thing I usually do but I have no strong opinion at all and we can use a single one if you prefer